vtex/faststore
# Chore/use generated schema envelop

## What's the purpose of this pull request?

Fixes GraphQL Schema Extensions. It also better organizes the generation and execution steps.

## How it works?

The GraphQL Schema Extensions were failing because the code was searching for the typeDefs during runtime and failed to do that. Pointing directly to the merged buil-time generated GraphQL schema solves the issue.

The GraphQL server now uses the merged build-time generated GraphqQL schema. This is good because we guarantee it will have the same definitions as the optimized persisted queries and it's less prone to runtime errors since the schema is already generated during build-time.

## How to test it?

Navigate to a PDP inside the deploy preview below. You should see an item id instead of the buy button.

### Starters Deploy Preview
